app/workers/audio_callback_worker.rb

```diff
@@ -41,6 +42,9 @@ def perform(_sqs_msg, job)
 
     Shoryuken.logger.info("Updating #{job['type']}[#{audio.id}]: status => #{audio.status}")
     audio.save!
+    # announce the audio changes using the story as resource.
+    announce(:audio, :update, id: audio.story.id, resource: audio.story)
```

Comment:
Is this how we're envisioning this? Or should the subject of the announce be the story?